packages/core/src/animation/AnimatorState.ts

```diff
@@ -83,6 +83,20 @@ export class AnimatorState {
     this._clipEndTime = Math.min(time, 1);
   }
 
+  /**
+   * The fixed start time of the clip, the range is 0 to clip.length, default is 0.
+   */
+  get clipStartFixedTime() {
+    return this._clipStartTime * this.clip.length;
+  }
+
+  /**
+   * The fixed end time of the clip, the range is 0 to clip.length, default is clip.length.
+   */
+  get clipEndFixedTime() {
+    return this._clipEndTime * this.clip.length;
+  }
+
```

Comment:
- Internal is better and fixed time is not good, maybe realTime?
- Add `clipEndTime` and `clipEndTime` return type